hardware/delays/MFA/MFA.py

```diff
@@ -109,9 +109,15 @@ def close(self):
 
     def get_position(self):
         # read
-        position = self.port.write(str(self.axis)+'TP', then_read=True)
-        # proccess (mm)
-        position = float(str(position).split('TP')[1])
+        while(True):
```

Comment:
@ksunden is it useful to comment this?

Reply:
yes, though I still feel this is a hack that I'd like to remove at some point if I can. But this works, thus I think it stays for now.